inputfiles/overridingTypes.jsonc

```diff
@@ -1589,6 +1589,7 @@
                         "generateKey": {
                             "name": "generateKey",
                             "additionalSignatures": [
+                                "generateKey(algorithm: \"Ed25519\", extractable: boolean, keyUsages: ReadonlyArray<Extract<KeyUsage, \"sign\" | \"verify\">>): Promise<CryptoKeyPair>",
```

Comment:
Isn't `Extract<KeyUsage, "sign" | "verify">` effectively just `"sign" | "verify"`? Maybe the latter can be concise.